app/models/poll.rb

```diff
@@ -5,7 +5,7 @@ class Poll < ApplicationRecord
   validates :title, presence: true
   validates :is_multiselect, inclusion: { in: [true, false] }
   validates :is_active, inclusion: { in: [true, false] }
-  validate :only_one_poll_active
+  #validate :only_one_poll_active
```

Comment:
why comment out and not delete? besides I am missing the test to ensure the wanted behavior

Reply:
This line could be removed, but I am not sure about the method executed there. What do you think?
Test will be added.

Reply:
remove them both

Reply:
Done.